sql/catalyst/pom.xml

```diff
@@ -39,6 +39,10 @@
       <groupId>org.scala-lang</groupId>
       <artifactId>scala-reflect</artifactId>
     </dependency>
+    <dependency>
```

Comment:
Some stuff like this became necessary due to other dependency changes and changes in their transitive dependencies.